requirements.txt

```diff
@@ -2,3 +2,5 @@ numpy
 pyyaml
 bitarray>=2.0.0
 six
+pyvisa
```

Comment:
Why did we not have this before @YannickDieter ? Was there a good reason?

Reply:
Same reason as #188 I would say.

Reply:
I would propose to add the requirements in this way, so that the install contains everything that is required for usage

Reply:
Let's move the discussion to #188. And @mjmucha Can you exclude pyvisa from the `requirements.txt` then? At last for now this keeps consistency.